src/Concerns/RememberChunkOffset.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Maatwebsite\Excel\Concerns;
+
+trait RememberChunkOffset
```

Comment:
Can you make this `RemembersChunkOffset` not the `s` as in `it remembers`

Reply:
Makes sense. fixed